Bump docs-theme-extras to v0.1.21, add pipefail to framework-tests, added descriptions (#36)
agentregistry-dev/website: content/docs/mcp/publish.md

---
title: Publish
weight: 20

> description: "Build your MCP server image and push it to agentregistry to share and deploy it anywhere."

Reviewer: ```suggestion
description: "Add an MCP server to the registry catalog. "
```
